src/sentry/relay/config.py

```diff
@@ -382,6 +382,7 @@ def get_transaction_metrics_settings(
     custom_tags = []
 
     if features.has("organizations:transaction-metrics-extraction", project.organization):
+        metrics.append("sentry.transactions.transaction.duration")
```

Comment:
shouldn't we add this metric to the file` sentry/sentry_metrics/sessions.py `and reuse it here?